### PR TITLE
Add missing things in core-js-pure 3

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
Fixes: #166

It seems that issues is not the optimization logic, but that we don't define the data of `corejs-pure`.

### CODE REVIEW

File: packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts [hunk 1]
```diff
@@ -355,7 +355,7 @@ export const StaticProperties: ObjectMap2<CoreJSPolyfillDescriptor> = {
   },
 
   ArrayBuffer: {
-    isView: define(null, ["es.array-buffer.is-view"]),
+    isView: define("array-buffer/is-view", ["es.array-buffer.is-view"]),
```
Comment (zloirock) [inline packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts:358]: It's missed in the pure version.

File: packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts [hunk 2]
```diff
@@ -434,10 +434,10 @@ export const StaticProperties: ObjectMap2<CoreJSPolyfillDescriptor> = {
   },
 
   Map: {
-    from: define(null, ["esnext.map.from", ...MapDependencies]),
-    groupBy: define(null, ["es.map.group-by", ...MapDependencies]),
-    keyBy: define(null, ["esnext.map.key-by", ...MapDependencies]),
-    of: define(null, ["esnext.map.of", ...MapDependencies]),
+    from: define("map/from", ["esnext.map.from", ...MapDependencies]),
```
Comment (zloirock) [inline packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts:437]: `.from` and `.of` are present in all collections.

**Comment by zloirock** [comment]
Yes, it looks like a partial workaround.

File: packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts [hunk 3]
```diff
@@ -515,17 +515,17 @@ export const StaticProperties: ObjectMap2<CoreJSPolyfillDescriptor> = {
...
-    any: define(null, [
+    any: define("promise/any", [
       "es.promise.any",
       "es.aggregate-error",
       ...PromiseDependenciesWithIterators,
     ]),
     race: define(null, PromiseDependenciesWithIterators),
-    try: define(null, ["esnext.promise.try", ...PromiseDependencies]),
+    try: define("promise/try", ["esnext.promise.try", ...PromiseDependencies]),
     withResolvers: define(null, [
```
Comment (zloirock) [inline packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts:529]: It's also a static method - it was added recently, but with `Map.groupBy` situation the same.

File: packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts (hunk 1 above)
Comment (liuxingbaoyu) [reply packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts:358]: image
Is it unavailable?

File: packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/built-in-static-methods/output.js [hunk 4]
```diff
@@ -193,3 +200,17 @@ Date.something;
...
+_Promise.all;
+_Promise$allSettled;
+_Promise$any;
+_Promise.race;
+_Promise.try;
+_Promise.withResolvers;
+_WeakMap.from;
+_ArrayBuffer$isView;
+Uint8Array.from;
+Uint8Array.of;
```
Comment (liuxingbaoyu) [inline packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/built-in-static-methods/output.js:216]: Is this expected? I didn't find the corresponding file.

File: packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts (hunk 1 above)
Comment (zloirock) [reply packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts:358]: It's a cap.

https://github.com/zloirock/core-js/blob/master/packages/core-js-pure/override/modules/es.array-buffer.is-view.js

File: packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/built-in-static-methods/output.js [hunk 5]
```diff
@@ -193,3 +199,17 @@ Date.something;
...
+_Map.from;
+_Map$groupBy;
+_Map.keyBy;
+_Map.of;
+_Promise.all;
+_Promise$allSettled;
+_Promise$any;
+_Promise.race;
+_Promise.try;
+_Promise.withResolvers;
```
Comment (liuxingbaoyu) [inline packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/built-in-static-methods/output.js:211]: This seems a bit strange.

**Review by nicolo-ribaudo — APPROVED** [review]

**Comment by nicolo-ribaudo** [comment]
Thank you!

**Comment by zloirock** [comment]
> `.from` and `.of` are present in all collections.

That means that those entries should be defined for all collections, not removed from `Map` -)

**Comment by liuxingbaoyu** [comment]
> > `.from` and `.of` are present in all collections.
> 
> That means that those entries should be defined for all collections, not removed from `Map` -)

Oops, I just tested that `Map.from/Map.of` does not exist.

**Pull request merged**